### DISCUSSION TITLE
Are we able to organize exported photos into folders that have the same name in our Albums?

### DISCUSSION THREAD

**Discussion by CarotMagic** [RhetTbull/osxphotos#1275] [Q&A] [answered]
So for example, I have photos in my Album named "Las Vegas Trip, Hawaii Trip, Landscape Shots, etc and would like to export it to their respective folders. How would one go about that?

**Comment by RhetTbull** [comment, marked as answer]
Use the option `--directory "{album}"` or `--directory "{folder_album}"` if you also want to include the folder path (and your albums are in folder)

**Reply by CarotMagic** [reply]
Thank You! That did it!
